api/v1alpha1/jindoruntime_types.go

```diff
@@ -170,6 +170,10 @@ type JindoRuntimeSpec struct {
 	// +kubebuilder:validation:Enum=HostNetwork;"";ContainerNetwork
 	// +optional
 	NetworkMode NetworkMode `json:"networkmode,omitempty"`
+
+	// Whether to enable hostPID or not
+	// +optional
+	DisableHostPID bool `json:"disableHostPID,omitempty"`
```

Comment:
Do you want to expose it to the enduser? May I know the reason?

Reply:
In some environment, it doesn't support hostPID

Reply:
I suggest to name it `hostPID` which is the same with Pod API.